lib/node_modules/@stdlib/blas/base/drot/README.md

```diff
@@ -0,0 +1,203 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2024 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# drot
+
+> Apply a plane rotation.
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var drot = require( '@stdlib/blas/base/drot' );
+```
+
+#### drot( N, x, strideX, y, strideY, c, s )
+
+Applies a plane rotation.
+
+```javascript
+var Float64Array = require( '@stdlib/array/float64' );
+
+var x = new Float64Array( [ 1.0, 2.0, 3.0, 4.0, 5.0 ] );
+var y = new Float64Array( [ 6.0, 7.0, 8.0, 9.0, 10.0 ] );
+
+drot( x.length, x, 1, y, 1, 1.0, 0.0 );
+// x => <Float64Array>[ 1.0, 2.0, 3.0, 4.0, 5.0 ]
+// y => <Float64Array>[ 6.0, 7.0, 8.0, 9.0, 10.0 ]
+```
+
+The function has the following parameters:
+
+-   **N**: number of indexed elements.
+-   **x**: first input [`Float64Array`][mdn-float64array].
+-   **strideX**: index increment for `x`.
+-   **y**: second input [`Float64Array`][mdn-float64array].
+-   **strideY**: index increment for `y`.
+-   **c**: double precision cosine.
```

Comment:
I was not able to find definitions for `c` and `s`. From where do these parameter descriptions derive?

Reply:
Okay. I find it. Let's make a change here.